### PR TITLE
Show prior number values as part of number collector

### PR DESCRIPTION
**Pull request by jgfoster**

### CODE REVIEW

File: src/TankControllerLib.cpp [hunk 1]
```diff
@@ -14,7 +14,8 @@
 #include "UIState/MainMenu.h"
 #include "UIState/UIState.h"
 
-const char TANK_CONTROLLER_VERSION[] = "0.3.0";
+// echo "#define GIT_HASH " `git rev-parse --short HEAD` > GitHash.h
```
Comment (jgfoster) [inline src/TankControllerLib.cpp:17]: remove me!

File: src/UIState/EnablePID.cpp [hunk 2]
```diff
@@ -7,6 +7,10 @@
 #include "Devices/LiquidCrystal_TC.h"
 #include "Devices/PHControl.h"
 
+float EnablePID::getPriorValue() {
```
Comment (jgfoster) [inline src/UIState/EnablePID.cpp:10]: Rename to "getCurrentValue"

File: src/UIState/NumberCollectorState.cpp [hunk 3]
```diff
@@ -55,13 +55,15 @@ void NumCollectorState::backSpace() {
...
 void NumCollectorState::printValue() {
   char strValue[16];
+  double prior = getPriorValue();
+
   if (!hasDecimal) {
-    snprintf(strValue, sizeof(strValue), "%.*f", 0, value);
+    snprintf(strValue, sizeof(strValue), "%.*f-> %i", getPriorValuePrecision(), prior, (int)value);
   } else if (factor == 10) {
-    snprintf(strValue, sizeof(strValue), "%.*f.", 0, value);
+    snprintf(strValue, sizeof(strValue), "%.*f-> %.*f.", getPriorValuePrecision(), prior, 0, value);
```
Comment (jgfoster) [inline src/UIState/NumberCollectorState.cpp:63]: Use %i here as well and add a comment explaining the decimal.

File: src/UIState/SetCalibrationClear.h [hunk 4]
```diff
@@ -10,10 +10,16 @@ class SetCalibrationClear : public UIState {
 public:
   SetCalibrationClear(TankControllerLib* tc) : UIState(tc) {
   }
+  virtual bool isInteger() {
```
Comment (jgfoster) [inline src/UIState/SetCalibrationClear.h:13]: No longer a subclass of NumberCollectorState.

File: src/UIState/TemperatureCalibration.h [hunk 5]
```diff
@@ -16,6 +16,9 @@ class TemperatureCalibration : public NumCollectorState {
   const char* name() {
     return "TemperatureCalibration";
   }
+  float getPriorValue() {
```
Comment (jgfoster) [inline src/UIState/TemperatureCalibration.h:19]: Should this also have a non-zero precision?

**Review by prestoncarman — APPROVED** [review]
LGTM pending requested changes.

**Pull request merged**